src/lib.ts

```diff
@@ -384,10 +375,20 @@ export function checkGithubVersion(): void {
     const mainBranch = gitBranches.find((branchInfo) => branchInfo.name === "main");
     const mainSha = mainBranch && mainBranch.commit ? mainBranch.commit.sha : "CustomBuild";
     if (process.env.GITHUB_SHA === mainSha) {
-      print("Garbo is up to date!", "blue");
+      print("Garbo is up to date!", HIGHLIGHT);
     } else {
       print("Garbo is out of date. Please run 'svn update!", "red");
       print(`${process.env.GITHUB_REPOSITORY}/main is at ${mainSha}`);
     }
   }
 }
+
+export function realmAvailable(
+  element: "spooky" | "stench" | "hot" | "cold" | "sleaze" | "fr"
+): boolean {
+  return get(`_${element}AirportToday`) || get(`${element}AirportAlways`);
+}
+
+export function fmt(num: number): string {
```

Comment:
i know what this does but have no idea what this name is meant to mean

Reply:
format?
either way, new name

Reply:
Fmt is pretty standard shortening for format. I can fix this but I prefer more concise names for things like this

Reply:
if it's standard, that seems fine. my brain just keeps trying to read it as `famweight` despite the lack of w lol